aws_s3_policies/aws_s3_bucket_public_read.py

```diff
@@ -6,9 +6,10 @@
 
 
 def policy(resource):
-    for grant in resource['Grants']:
-        if grant['Grantee']['URI'] in GRANTEES and grant[
-                'Permission'] in PERMISSIONS:
-            return False
+    if 'Grants' in resource and resource['Grants'] is not None:
```

Comment:
This could potentially be simplified to:
```
if resource.get('Grants', {}): 
```

Reply:
Yep, like that way also